dashboard/test/ui/features/teacher_tools/pl_sections.feature

```diff
@@ -22,21 +20,20 @@ Feature: Professional learning Sections
     When I select facilitator participant type
 
     # Edit Section Form
```

Comment:
nit: this isn't the edit section form anymore -- can you update these comments?

Reply:
Updated- thanks!